Fix typo in README (#9)
atx/wtype: README.md

## Limitations

> To support arbitrary unicode characters, `wtype` generates a keymap on the fly and submits it to the compositor for distribution. Unfortunately this is broken under Xwayland as X does have an upper limit on the amount of defined keycode-keysym pairs.

Author: Maybe even

```suggestion
To support arbitrary unicode characters, `wtype` generates a keymap on the fly and submits it to the compositor for distribution. Unfortunately this is broken under Xwayland as X has an upper limit on the amount of defined keycode-keysym pairs.
```

?
